### PR TITLE
Generate primes

### LINKED ISSUE #10: Natively generate two big prime numbers
Currently, the CLI requires the user to supply in their own prime numbers however they generate them:
```
  -p prime prime, --primes prime prime
                        Load from two big prime numbers
```
Having the CLI generate them natively would be nice.

### PR DESCRIPTION
**Pull request by LyndonFan**
# Description

Please include a general summary of the change and/or which issue it fixed. 

Fixes #10 

## Type of change 
Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Have you tested your changes
Yes, manually tried out different combinations of input.

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Note
The generation requires the cryptodome library. I installed it with ``pip install pycryptodomex``.
See https://pycryptodome.readthedocs.io/en/latest/src/installation.html

### CODE REVIEW

File: rsactf/rsa.py [hunk 1]
```diff
@@ -1,19 +1,28 @@
 import argparse
 import math
 import sys
+from Cryptodome.Util import number
```
Comment (alanyee) [inline rsactf/rsa.py:4]: Could you add the library in the `setup.py`?

File: README.md [hunk 2]
```diff
@@ -4,16 +4,21 @@ Generate textbook integer-type RSA schema for fun/[CTF](https://ctftime.org/)s.
 ## usage
 
 ```
-usage: rsa-keygen [-h] [-m MESSAGE] [-p prime prime] [-e exponent]
+usage: rsa.py [-h] [-m MESSAGE] [-l prime_length] [-ls prime_lengths prime_lengths]
```
Comment (alanyee) [inline README.md:7]: I would like to maintain the `rsa-keygen` in the usage statement.

**Comment by LyndonFan** [comment]
I've reversed my changes in the README.md, and added the package (manually) in setup.py

**Comment by alanyee** [comment]
I can merge it from here. The linter was recently added, so it is tripping up on code that was written by me, not you, so it should be out of scope for this PR. I have tested this PR myself, and it is working. Thank you!

**Pull request merged**